Sources/MonkeyKing.swift

```diff
@@ -1184,3 +1190,33 @@ extension MonkeyKing {
         }
     }
 }
+
+extension MonkeyKing {
+    public enum Program {
+        public enum WeChatSubType {
+            case miniApp(userName: String, path: String?, type: MiniAppType)
+        }
+
+        case weChat(WeChatSubType)
+    }
+
+    public class func launch(program: Program, completionHandler: @escaping LaunchCompletionHandler) {
+        guard let account = shared.accountSet[.weChat] else {
+            completionHandler(.failure(.noAccount))
+            return
+        }
+
+        shared.launchCompletionHandler = completionHandler
+
+        switch program {
+        case .weChat(let type):
+            switch type {
+            case .miniApp(let userName, let path, let type):
+                openURL(urlString: "weixin://app/\(account.appID)/jumpWxa/?userName=\(userName)&path=\(path ?? "")&miniProgramType=\(type.rawValue)") { flag in
```

Comment:
If the `path` can be nil, use URLComponents to generate the url is better

Reply:
Sure, I'll fix it.